torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -245,6 +280,26 @@ def should_skip_lowering(op: torch.fx.node.Node, qconfig_map: Dict[str, QConfigA
     torch._ops.ops.quantized.conv3d_prepack,
 }
 
+# Mapping from a functional to (activation_compute_dtype, weight_dtype) to lower to a 2-tuple of
+#   1) The dynamically quantized version of the op
+#   2) The dynamically quantized version of the op fused with relu, if it exists, else None
```

Comment:
Looks like this comment is out of date now? Or maybe the wording is confusing, maybe something like
```
Mapping from a functional to a dictionary, where the key is a 2-tuple of (...)
and the value is a 2-tuple of (...)
```

Reply:
sounds good, yeah I can change the wording, thanks